projects/ngu-carousel/src/lib/ngu-carousel/ngu-carousel.component.ts

```diff
@@ -59,12 +59,26 @@ export class NguCarousel<T> extends NguCarouselStore
   private withAnim = true;
   activePoint: number;
   isHovered = false;
+  private inputs: NguCarouselConfig;
+  // tslint:disable-next-line:no-input-rename
+  @Input('inputs')
+  set publicInputs(inputs: NguCarouselConfig) {
+    this.inputs = inputs;
+  }
+  private carouselLoad = new EventEmitter<any>();
+  // tslint:disable-next-line:no-input-rename
+  @Output('carouselLoad')
```

Comment:
@Output() carouselLoad = new EventEmitter();

Reply:
yes this would work as well